services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -668,6 +693,41 @@ class ProtectedPathSpec extends BaseSpecification {
 
         currentLastUpdate < northbound.getFlow(flow.id).lastUpdated
 
+        and: "New meter is created on the src and dst switches"
+        def newSrcSwitchCreatedMeterIds = getCreatedMeterIds(srcSwitch.dpId)
+        def newDstSwitchCreatedMeterIds = getCreatedMeterIds(dstSwitch.dpId)
+        newSrcSwitchCreatedMeterIds != srcSwitchCreatedMeterIds
```

Comment:
List is an **ordered** collection. So `assert [1,2] != [2,1]` won't fail. This verification is dangerous. Call `.sort()` on both sides or use `containsAll` or sth else

Reply:
fixed